packages/core/android/src/test/kotlin/com/datadog/reactnative/DdSdkTest.kt

```diff
@@ -1263,6 +1281,40 @@ internal class DdSdkTest {
         }
     }
 
+    @Test
+    fun `𝕄 send long tasks 𝕎 frame time is over threshold() {}`(
+        @LongForgery timestampNs: Long,
+        @LongForgery(ONE_HUNDRED_MILLISSECOND_NS, ONE_SECOND_NS) frameDurationNs: Long,
```

Comment:
It is better to use named arguments.

```suggestion
        @LongForgery(min = ONE_HUNDRED_MILLISSECOND_NS, max = ONE_SECOND_NS) frameDurationNs: Long,
```